### ISSUE TITLE
Migrate from sourceforge.net to GitHub (ghcr.io or raw.githubusercontent.com)

### ISSUE THREAD

**Issue by imuhdork** [swig/swig#2892]
Sourceforge.net and its mirrors currently host the swig brew formula and packages. Unfortunately, many organizations are blocking any downloads from sourceforge.net given various security concerns with the practices in that platform.

Many other packages and brew formula (esepcially) have migrated to the GitHub container registry (ghcr.io) or accessing the raw  files directly from GitHub via raw.githubusercontent.com. Both of these options appear to be an industry trend with greater trust of the content. Please consider migrating where the swig package and formulae can be downloaded from Sourceforge to GitHub.

**Comment by ojwb** [comment]
I don't think we maintain the swig brew formula - that seems to be in the homebrew-core repo: https://github.com/Homebrew/homebrew-core/blob/0a816916dd912a9c7c3835cb0615ad6a0a24cc59/Formula/s/swig.rb

**Comment by imuhdork** [comment]
Could the brew formula you linked just be pointing to where the swig packages appear to be uploaded as defined in [https://github.com/swig/swig/blob/master/Tools/mkrelease.py](https://github.com/swig/swig/blob/master/Tools/mkrelease.py)? It looks like swig packages are available from sourceforge in general and brew formula are just using just using that package location.

**Comment by ojwb** [comment]
Our website is hosted on sourceforge currently, but that isn't what you asked us to change.

Moving to the appropriate tracker.

> Unfortunately, many organizations are blocking any downloads from sourceforge.net given various security concerns with the practices in that platform.

Can you provide evidence for this claim?
